packages/components/src/Grid/GridCell.tsx

```diff
@@ -0,0 +1,16 @@
+// Only need the component to pass in the data as a react children
+
+import { useAssert } from "@jobber/hooks/useAssert";
+import React, { PropsWithChildren } from "react";
+import { GridCellProps } from "./InternalGridCell";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function GridCell(props: PropsWithChildren<GridCellProps>) {
```

Comment:
underscore did not work for this eh?

```suggestion
export function GridCell(_: Props>) {
```